### PR TITLE
Deep Learning Fundamentals -- Learning Deep Learning With A Modern Open Source Stack

### PR DESCRIPTION
**Pull request by rasbt**
### Description

This course has many thousands students and got a lot of positive feedback, and I thought it would be nice adding it. There are >15 hours of materials with hundreds of videos (5-15 min in length).

It covers everything from the ground up, from backpropagation to finetuning LLMs. What makes this course unique is that it goes beyond the topics covered in typical introductory courses and colleges, covering topics such as

- multi-GPU and mixed precision training
- learning rate schedulers
- unsupervised pretraining and self-supervised learning, and many more.

It's also 100% free and also features quizzes, code materials, and optional code exercises.

### Pull request checklist

- [x] Changes added to [data.csv](https://github.com/SkalskiP/courses/blob/master/automation/data.csv). Read contribution [guidelines](https://github.com/SkalskiP/courses/blob/master/CONTRIBUTING.md) to learn more.
- [x] Changes added to [README.md](https://github.com/SkalskiP/courses/blob/master/automation/data.csv) Read contribution [guidelines](https://github.com/SkalskiP/courses/blob/master/CONTRIBUTING.md) to learn more.
- [x] Pull request description added

### CODE REVIEW

**Comment by SkalskiP** [comment]
Thanks a lot 🙏🏻 ! Looks awesome. 

by the way sorry, but I couldn't stop myself from tweeting about it :)
